tests/pre-command.bats

```diff
@@ -46,6 +46,20 @@ load '/usr/local/lib/bats/load.bash'
   unstub aws
 }
 
+@test "Syncs with cache control metadata" {
+  export BUILDKITE_PLUGIN_AWS_S3_SYNC_SOURCE=s3://source
+  export BUILDKITE_PLUGIN_AWS_S3_SYNC_DESTINATION=destination/
+  export BUILDKITE_PLUGIN_AWS_S3_SYNC_CACHE_CONTROL="public,max-age=315360000"
+
+  stub aws "s3 sync --cache-control=public,max-age=315360000 s3://source destination/ : echo s3 sync --cache-control=public,max-age=315360000"
```

Comment:
This combination looks a little weird. Is there a use for setting cache-control when downloading files? 🤷

Reply:
No there isn't. Sorry this was an oversight on my behalf, as I was unclear on the difference between the pre and post commands. Now that I have a better understanding I can see why this is confusing :)